public/data/nikhil96.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Nikhil Sharma",
+  "type": "personal",
+  "bio": "Learning to contribute!",
+  "avatar": "https://github.com/itsmenikhill.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/itsmenikhill",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/itsnikhilpeeps",
```

Comment:
Check whether there is a typo in your username in the link because it's a link to an account that doesn't exist! 

**Update it with the correct one!** 🛠

![image](https://user-images.githubusercontent.com/88786642/185542246-20a51967-4448-4fa9-86ae-d4bc101ac1ac.png)


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I changed my twitter handle after I made changes to the file. Thanks for pointing it out.